### DISCUSSION TITLE
Early stopping in NP training

### DISCUSSION THREAD

**Discussion by Amjad-AbuRmileh** [ourownstory/neural_prophet#452] [Q&A - get help using NeuralProphet]
Is there an option to configure early stopping parameters for the neural prophet?

**Comment by ourownstory** [comment]
Hi @Amjad-AbuRmileh 
If you want to speed up training, you can play with epochs, batch and lr, those are more effective with the one cycle policy than early stopping.

**Reply by Amjad-AbuRmileh** [reply]
I didn't understand the one-cycle policy thing?

**Reply by ourownstory** [reply]
we use this:

https://arxiv.org/abs/1803.09820

explanation:
https://towardsdatascience.com/finding-good-learning-rate-and-the-one-cycle-policy-7159fe1db5d6
